PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+TO CONTRIBUTORS:
+
+Make sure you have:
+
+ * [ ] reviewed and applied the coding style: [coding](./docs/style.md), [naming](./docs/naming.md)
+ * [ ] for tactics:
+     * [ ] added or adapted documentation in [tactics.md](./docs/tactics.md)
+     * [ ] write an example of use of the new feature in [tactics.lean](./tests/tactics.lean)
+  * [ ] reviewed other PRs to avoid redundancy or overlap
```

Comment:
I don't think we need to ask contributors to avoid redundancy with other PRs, that has not been a problem in the past. More important is redundancy with mathlib itself. Another way to put this is to make sure that you have put the theorem in the right place, and make sure that mathlib currently has a gap there.